### PR TITLE
Tag the EBS volumes.

### PR DESCRIPTION
**Pull request by pagreene**
When tagging batch instances, also tag the EBS volumes.

### CODE REVIEW

File: indra/util/aws.py [hunk 1]
```diff
@@ -99,6 +99,10 @@ def tag_instance(instance_id, **tags):
         logger.info('Adding project tags "%s" to instance %s'
                     % (filtered_tags, instance_id))
         instance.create_tags(Tags=tag_list)
+        vols = instance.volumes.all()
+        for page in vols.pages():
+            for vol in page:
+                vol.create_tages(Tags=tag_list)
```
Comment (bgyori) [inline indra/util/aws.py]: Isn't this supposed to be `create_tags`?

Comment (pagreene) [reply indra/util/aws.py]: Indeed. For some reason I make that typo a lot.

File: indra/util/aws.py [hunk 2]
```diff
@@ -99,6 +99,10 @@ def tag_instance(instance_id, **tags):
         logger.info('Adding project tags "%s" to instance %s'
                     % (filtered_tags, instance_id))
         instance.create_tags(Tags=tag_list)
+        vols = instance.volumes.all()
+        for page in vols.pages():
```
Comment (jmuhlich) [inline indra/util/aws.py]: Can't this be simplified to `for vol in vols` and omit the explicit page traversal? Or even combine it with the previous line, `for vol in instance.volumes.all():`

Comment (pagreene) [reply indra/util/aws.py]: Possibly. This is just how I got it to work in ipython. `vols` didn't seem to be iterable, although I suppose I didn't try just iterating over it. But the only thing that returned an apparent iterable was `vols.pages()`. It seemed obtuse, but it worked. :man_shrugging:

**Pull request merged**